.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,58 @@
+---
+name: "🐞 Bug report"
+about: Report a bug
+---
+<!--
+
+Hello.
+
+Please search open and closed issues before submitting.
+
+Existing issues often contain information about workarounds, resolution, or progress updates.
+
+Thanks for your help.
+
+-->
+
+
+### Environment
+
+<!-- OS version
+# macOS
+❯ sw_vers
+
+# linux
+❯ lsb_release -a
```

Comment:
`lsb_release -a` and `systeminfo` not tested, I don’t have a windows or linux machine handy.

Reply:
Also we only need the first 4 or so lines of `systeminfo` if anybody knows how to pipe it into the windows equivalent of `head`.